homeassistant/components/unifi_access/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "unifi_access",
+  "name": "UniFi Access",
+  "codeowners": ["@hagen93"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/unifi_access",
+  "homekit": {},
+  "iot_class": "local_push",
+  "requirements": ["uiaccessclient==0.9.1"],
+  "ssdp": [],
+  "zeroconf": []
+}
```

Comment:
Quality scale should be set to bronze and the quality_scale.yaml should be evaluated

Reply:
Absolutely. About evaluating the quality scale; do I manually evaluate each criteria, or has this been automated and will be generated by tooling?

Reply:
A part is automated, but I strongly recommend reading the documentation to see what we do and why we do what we do